QUICKSTART-VMWARE.md

```diff
@@ -0,0 +1,248 @@
+# Using Bottlerocket as a Kubernetes worker node with VMware
+
+> The [VMware variant](variants/README.md#vmware-k8s-120-vmware-kubernetes-120-node) is currently in preview phase and we're looking for [feedback](https://github.com/bottlerocket-os/bottlerocket#contact-us).
+> Please try it out and let us know what you think!
+
+This quickstart will walk through joining a Bottlerocket VM guest to an existing Kubernetes cluster running in VMware.
+
+## Prerequisites
+
+You must be able to access vSphere, via GUI or some type of client.
+We will use the CLI tool [`govc`](https://github.com/vmware/govmomi/tree/master/govc) to communicate with vSphere in this quide.
+`govc` can use [environment variables or take arguments](https://github.com/vmware/govmomi/tree/master/govc#usage) to specify needed parameters.
+For the purposes of this guide we will assume that the following environment variables are set to the proper values in your environment:
+```
+GOVC_URL
+GOVC_USERNAME
+GOVC_PASSWORD
+GOVC DATACENTER
+GOVC_DATASTORE
+GOVC_NETWORK
+GOVC_RESOURCE_POOL
+GOVC_FOLDER
+```
+
+This guide assumes you already have a functioning Kubernetes cluster running in VMware.
+You'll need to have [`kubectl`](https://kubernetes.io/docs/tasks/tools/#kubectl) and [`kubeadm`](https://kubernetes.io/docs/tasks/tools/#kubeadm) installed, as well as a `kubeconfig` for your cluster.
+These tools allow us to access information about your cluster to facilitate the joining of Bottlerocket nodes.
+
+You'll need to install [`tuftool`](https://github.com/awslabs/tough/blob/develop/tuftool/README.md) to assist you with fetching the signed Bottlerocket OVA from the Bottlerocket TUF repository.
+We're working on making this a better experience for users that do not want to install `tuftool`.
+
+## Fetch the OVA
+
+The Bottlerocket OVA is signed and uploaded alongside the rest of the Bottlerocket release artifacts.
+
+You first need the Bottlerocket root role, which is used by `tuftool` to verify the OVA.
+The following will download and verify the root role itself:
+
+```
+curl -O "https://cache.bottlerocket.aws/root.json"
+sha512sum -c <<<"90393204232a1ad6b0a45528b1f7df1a3e37493b1e05b1c149f081849a292c8dafb4ea5f7ee17bcc664e35f66e37e4cfa4aae9de7a2a28aa31ae6ac3d9bea4d5  root.json"
+```
+
+Next, set your desired version and variant, and download the OVA:
+```
+VERSION="v1.1.0"
+VARIANT="vmware-k8s-1.20"
+
+tuftool download . --target-name bottlerocket-${VARIANT}-x86_64-${VERSION}.ova \
+   --root ./root.json \
+   --metadata-url "https://updates.bottlerocket.aws/2020-07-07/${VARIANT}/x86_64/" \
+   --targets-url "https://updates.bottlerocket.aws/targets/"
+```
+
+## Upload the OVA
+
+Once you have downloaded the OVA, you can upload it to vSphere.
+
+The first command generates a spec file (`bottlerocket_spec.json` in this case) using the OVA and gives you few options for your deployment.
+```
+govc import.spec <Bottlerocket OVA > bottlerocket_spec.json
```

Comment:
For consistency maybe `govc import.spec <bottlerocket...ova> > bottlerocket_spec.json`